src/main/java/mate/hwdao/dao/impl/jdbc/ManufactureDaoJdbcImpl.java

```diff
@@ -0,0 +1,124 @@
+package mate.hwdao.dao.impl.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.hwdao.dao.ManufacturerDao;
+import mate.hwdao.lib.Dao;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.util.ConnectionUtil;
+
+@Dao
+public class ManufactureDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try {
+            String query = "INSERT INTO manufacture (name, country, isexist) VALUES (?, ?, ?)";
```

Comment:
move query outside try block. 
isexist is not a correct naming - is_deleted / exists
We don't need to add it to query, just make a default value in the db, so it will be put automatically

Reply:
done